L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker_cfg.py

```diff
@@ -128,7 +127,6 @@
     AssociationName = "l1tTrackVertexNNAssociationProducer"
 else:
     process.l1tVertexFinderEmulator = process.l1tVertexProducer.clone()
```

Comment:
Could this line be removed as well? if the import worked, `process.l1tVertexFinderEmulator` will already exist